acp/wpn_acp_module.php

```diff
@@ -105,6 +105,13 @@ public function display_settings()
 			'S_WEBPUSH_METHOD_ENABLED' 		=> $this->config['wpn_webpush_method_enabled'],
 			'U_ACTION'						=> $this->u_action,
 		]);
+
+		if (!$this->request->server('HTTPS', false))
```

Comment:
Currently it shows error on localhost. Probably it's worth adding a check for `!$this->request->server('SERVER_NAME', '') == 'localhost')` or some kind of that.

Reply:
oh yeah